docs/source/recipes/developing-a-project.md

```diff
@@ -0,0 +1,32 @@
+---
+myst:
+  html_meta:
+    "description": "Developing a Volto project"
+    "property=og:description": "Developing a Volto project"
+    "property=og:title": "AppExtras component"
+    "keywords": "Volto, Plone, frontend, React, app extra component"
+---
+
+(frontend-developing-a-project-label)=
+
+# Developing a project
+
+When you start developing a Plone project, it is recommended that you use Plone's cookiecutter boilerplate generator.
+See {ref}`install-packages-1-label` for more information.
```

Comment:
This syntax is actually more correct because the Volto docs itself does not have an internal reference to Plone 6 docs until it gets included as a submodule. This syntax uses Intersphinx.

```suggestion
See {doc}`plone:install/install-from-packages` for more information.
```